README.md

```diff
@@ -69,6 +72,8 @@ $ npm run test:cov
 
 Nest is an MIT-licensed open source project. It can grow thanks to the sponsors and support by the amazing backers. If you'd like to join them, please [read more here](https://docs.nestjs.com/support).
 
+
+
```

Comment:
Not sure these extra spaces are needed?

Reply:
So sorry this might be a mistake from my side